docs/dev.rst

```diff
@@ -1,20 +1,570 @@
-Develop a project
+Develop a Project
 =================
 
-Set up a dev org
+A general overview on how to develop a Salesforce project with CumulusCI.
+
+
+
+Set Up a Dev Org
 ----------------
 
-Track and retrieve changes
---------------------------
+The ``dev_org`` flow creates an org to develop on by moving all metadata (managed and unmanaged) into the org, and configuring it to be ready for development.
+
+.. note:: Run ``cci flow info dev_org`` for a full list of the ``dev_org`` flow steps.
+
+To run the ``dev_org`` flow against the project's `default org<TODO>`_:
+
+.. code-block:: console
+
+    $ cci flow run dev_org
+
+To explictly list the org ``<org_name>`` against the dev_org flow, use the ``--org`` option.
+
+    Example: Run the ``dev_org`` flow against the org currently defined as ``dev`` in CumulusCI.
+
+.. code-block:: console
+
+    $ cci flow run dev_org --org dev
+
+..
+
+    Open the new ``dev`` org to begin development.
+
+.. code-block:: console
+
+    $ cci org browser dev
+
+
+Set the Capture State
+^^^^^^^^^^^^^^^^^^^^^
+
+To make changes to capture in an org, use the ``snapshot_changes`` task to create a snapshot. A snapshot tells the Salesforce CLI source tracking to set the org's current state as a baseline for changes to be made against.
```

Comment:
@Br4nd0R Mark this for discussion before we merge.

Reply:
Now that David brings this up, I realize he's right, we shouldn't need to include content on this command. It made it here because it is currently included in the original doc set. I think this is a good candidate for content that needs not be carried over to the new docs.